react/features/display-name/components/web/DisplayName.tsx

```diff
@@ -336,4 +195,4 @@ function _mapStateToProps(state: IState, ownProps: Partial<Props>) {
     };
```

Comment:
maybe get rid of _mapStateToProps  - refactor to useSelector in the component- so then we can remove the connect(...